src/components/ProductCard.js

```diff
@@ -0,0 +1,20 @@
+import "./ProductCard.css";
+import Niacinamide from "../images/niacinamide-bottle.png";
+
+export default function ProductCard() {
+  return (
+    <div className="ProductCard">
+      <h5 className="ProductCard__headline">Niacinamide 10% + Zinc 1%</h5>
+      <img src={Niacinamide} alt="product bottle" className="bottle-image" />
```

Comment:
the `alt` is for screen readers so you might wanna be more specific for blind customers wanting Niacinamide..